launchdarkly/config.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	AgentVersion = "1.2.1"
+	AgentVersion = "1.2.2"
```

Comment:
i don't know what this does but this is on purpose right?

Reply:
Yup that sets the User Agent for all API calls. We should always try to keep this synced up with the version.